src/Uno.AzureDevOps/Uno.AzureDevOps.Shared/Framework/Tasks/TaskNotifier.cs

```diff
@@ -60,6 +63,12 @@ private void RunTask(Task taskToExecute)
 			taskToExecute.ContinueWith(
 				task =>
 				{
+#if !__WASM__
+					if (Connectivity.NetworkAccess != NetworkAccess.Internet)
```

Comment:
This takes into account both Wifi and Cellular network, right?

Reply:
Yes it is !